package.json

```diff
@@ -24,19 +24,23 @@
   "exports": {
     ".": {
       "require": "./dist/index.js",
-      "default": "./index.mjs"
+      "default": "./index.mjs",
+      "types": "./dist/index.d.ts"
```

Comment:
```suggestion
      "types": "./dist/index.d.ts",
      "default": "./index.mjs"
```

Reply:
The `types` must come first as the first key. This is some weird requirement by a bunch of tools.